pagopa/pn-frontend
# Pn 7637 fe mittente filtro per codice iun

## Short description
Added paste event control in filters for IUN and Fiscal Code / Tax ID.

## List of changes proposed in this pull request
- Added paste event for PA, PG and PF

## How to test
Open a detailed notification, then try to copy a IUN code by point the mouse a bit outside the row where IUN is located to get the whitespaces too. Then paste in filter, now should be fine.
